app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -37,6 +37,10 @@ def supports_port?
     true
   end
 
+  def supports_api_version?
+    true
```

Comment:
Does this need to do an actual check here?
I'm guessing it is ok, but just wanted to ask.